app/src/main/java/com/sunasterisk/weather/data/source/WeatherDataSource.kt

```diff
@@ -0,0 +1,12 @@
+package com.sunasterisk.weather.data.source
+
+interface WeatherDataSource {
+
+    interface Local {
+
+    }
```

Comment:
khi nào dùng thì thêm { } vào e
```suggestion
    interface Local
```